ERROR - 2021-07-19 14:02:33 --> Table 'ci4.loket' doesn't exist
CRITICAL - 2021-07-19 14:02:33 --> Call to a member function getResultArray() on bool
#0 C:\xampp\htdocs\ci\ci\app\Controllers\Loket.php(25): App\Models\LoketModel->getLoket()
#1 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(928): App\Controllers\Loket->index()
#2 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Loket))
#3 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci\ci\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-07-19 14:02:59 --> Table 'ci4.loket' doesn't exist
CRITICAL - 2021-07-19 14:02:59 --> Call to a member function getResultArray() on bool
#0 C:\xampp\htdocs\ci\ci\app\Controllers\Loket.php(25): App\Models\LoketModel->getLoket()
#1 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(928): App\Controllers\Loket->index()
#2 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Loket))
#3 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci\ci\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-07-19 14:03:22 --> Table 'ci4.loket' doesn't exist
CRITICAL - 2021-07-19 14:03:22 --> Call to a member function getResultArray() on bool
#0 C:\xampp\htdocs\ci\ci\app\Controllers\Loket.php(25): App\Models\LoketModel->getLoket()
#1 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(928): App\Controllers\Loket->index()
#2 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Loket))
#3 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci\ci\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-07-19 14:03:49 --> Table 'ci4.loket' doesn't exist
CRITICAL - 2021-07-19 14:03:49 --> Call to a member function getResultArray() on bool
#0 C:\xampp\htdocs\ci\ci\app\Controllers\Loket.php(25): App\Models\LoketModel->getLoket()
#1 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(928): App\Controllers\Loket->index()
#2 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Loket))
#3 C:\xampp\htdocs\ci\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci\ci\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
